pysb/bng.py

```diff
@@ -116,6 +119,11 @@ def __enter__(self):
     def __exit__(self):
         return
 
+    def _delete_tmpdir(self):
+        for f in os.listdir(self.base_directory):
+            os.unlink(os.path.join(self.base_directory, f))
+        os.rmdir(self.base_directory)
+
```

Comment:
I can't see why we wouldn't be clear to blow a temp working dir away. If the user wants to keep the temp output files around, they need to pass `cleanup=False`.